bpf/custom/README.rst

```diff
@@ -0,0 +1,125 @@
+..  Copyright (C) 2021 Authors of Cilium
+
+===================================
+Tail Call Hooks for Custom Programs
+===================================
+
+The files in this directory implement a sample byte counter program which can
+be attached to dedicated hooks in the datapath.
+
+.. note::
+    This is a beta feature. Please provide feedback and file a GitHub issue if
+    you experience any problems.
```

Comment:
FYI, I think this syntax is specific to Read the Docs. GitHub won't render it: https://gist.github.com/pchaigno/9397bdad9553204f93fee0226d384f1e.

Reply:
Right, I suspected so but forgot to check, thank you. FYI it is part of the directives [documented](https://docutils.sourceforge.io/docs/ref/rst/directives.html#attention) and implemented by docutils, which also defines the specifications for RST, but indeed not rendered on GitHub. I'll fix.